plugins/table.md

```diff
@@ -80,6 +80,26 @@ tinymce.init({
 });
 ```
 
+### `table_use_colgroups`
+
+This option allows you to set the default type of new tables created by the table plugin to use colgroups. Tables inserted by other means, or already existing tables, will not be changed.
```

Comment:
This doesn't read very well, as it mentions "default type" but then that isn't explaining that it's for `colgroups` until the end of the sentence. Maybe something like this instead?
```suggestion
This option allows you to set whether `colgroup`s should be added to new tables created by the table plugin. Tables inserted by other means, or already existing tables, will not be changed.
```

Reply:
Seems quite sensible. I'll make the change.

Reply:
fixed in https://github.com/tinymce/tinymce-docs/pull/1679/commits/dcb92c09d3fbeffeeabe745dd5699886454b2523